### PR TITLE
Styles applied with Bootstrap and Sass

### PR DESCRIPTION
**Pull request by miguel0577**
## What?
Visual environments of the main page and creation articles were improved with the help of Bootstrap and Sass

## Why?
Having a visually pleasing interactivity environment helps the user to be able to enter the necessary data in a better way.

## How?
Bootstrap was installed in the Ruby project and through templates and sheet styles in SASS the visual environments were modified

## Tests?
On the local server, the project works perfectly and is visually pleasing

## Screenshots
image
image

## Anything else?
Upcoming visual styles will be limited to changes in blog functionality

### CODE REVIEW

File: app/views/articles/_form.html.erb [hunk 1]
```diff
@@ -1,26 +1,57 @@
...
+        </div>
+
+        <div class="label-submit-article">
+          <%= form.submit %>
+        </div>
+      <% end %>
+         
+    </main>
+
+    <footer>
```
Comment (estebancordoba) [inline app/views/articles/_form.html.erb]: I see this repeated, create a partial or add the code to the main template

File: config/importmap.rb [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
 
 # Pin npm packages by running ./bin/importmap
-
 pin 'application', preload: true
 pin '@hotwired/turbo-rails', to: 'turbo.min.js', preload: true
 pin '@hotwired/stimulus', to: 'stimulus.min.js', preload: true
 pin '@hotwired/stimulus-loading', to: 'stimulus-loading.js', preload: true
 pin_all_from 'app/javascript/controllers', under: 'controllers'
+pin 'bootstrap', to: 'bootstrap.min.js', preload: true
+pin '@popperjs/core', to: 'popper.js', preload: true
```
Comment (estebancordoba) [inline config/importmap.rb:10]: why is this line necessary?

**Review by estebancordoba — CHANGES_REQUESTED** [review]

File: config/importmap.rb (hunk 2 above)
Comment (miguel0577) [reply config/importmap.rb:10]: Bootstrap tooltips and popovers depend on the popper.js file for positioning, also the bootstrap gem already depends on the popper_js gem, so these tools must be pinned to the importmap file in the config path

File: app/views/articles/_form.html.erb (hunk 1 above)
Comment (miguel0577) [reply app/views/articles/_form.html.erb]: Understood, in pull request #14 I will add these changes

Comment (estebancordoba) [reply app/views/articles/_form.html.erb]: Could you add that change in this PR?

Comment (miguel0577) [reply app/views/articles/_form.html.erb]: Sure, I'll add the change to this pull request.

**Review by miguel0577 — COMMENTED** [review]
Changes made in this pull request!

**Review by estebancordoba — APPROVED** [review]
